build.cake

```diff
@@ -409,7 +409,7 @@ private void SignBinaries(string path)
 
 	Sign(files, new SignToolSignSettings {
 			ToolPath = MakeAbsolute(File("./certificates/signtool.exe")),
-            TimeStampUri = new Uri("http://timestamp.verisign.com/scripts/timstamp.dll"),
+            TimeStampUri = new Uri("http://timestamp.digicert.com"),
```

Comment:
Why did this change?

Reply:
That url no longer works. Apparently Verisign shut it down after it sold some part of the business to Semantic: https://stackoverflow.com/questions/65541786/is-the-verisign-timestamp-server-down